src/system/SystemConfig.h

```diff
@@ -111,6 +111,14 @@
 
 // clang-format off
 
+#ifndef CHIP_SYSTEM_CONFIG_USE_IO_THREAD
```

Comment:
See prior comments about CHIP_SYSTEM_CONFIG_USE_IO_THREAD. There might be an opportunity to detect and flag incorrect configurations.